README.md

```diff
@@ -87,12 +87,11 @@ There is an official image, built from source at: https://hub.docker.com/r/nofis
 
 ### Install from source
 
- - `wget https://github.com/HelloZeroNet/ZeroNet/archive/py3/ZeroNet-py3.tar.gz`
- - `tar xvpfz ZeroNet-py3.tar.gz`
- - `cd ZeroNet-py3`
- - `sudo apt-get update`
- - `sudo apt-get install python3-pip`
- - `sudo python3 -m pip install -r requirements.txt`
+ - `sudo su`
```

Comment:
Please also add `exit` command to the end if you switch to `sudo su`. Some users who don't know Linux well might continue working under root and accidentally break something.